api/net/appsettings.json

```diff
@@ -71,6 +72,13 @@
       "MessageSendMaxRetries": 10000000,
       "BatchSize": 16384,
       "LingerMs": 1
+    },
+    "Consumer": {
+      "GroupId": "API",
```

Comment:
This value needs to be overridden in Openshift with a dynamic statefulset variable.